RDF_Tests/ToTTLTests.cs

```diff
@@ -384,6 +388,58 @@ public static void NurbsCurve_ControlPoints()
             Assert.IsTTLParsable(TTLGraph);
         }
 
-   
+        [Test]
+        public static void UriGuidBHoMObj()
+        {
+            Room room = new Room();
+            room.Perimeter = new Polyline() { ControlPoints = new List<Point>() { new Point(), new Point() { X = 5 }, new Point() { X = 99 } } };
+            room.Location = new Point();
+            room.Name = "first room object";
+
+            CSharpGraph cSharpGraph = Compute.CSharpGraph(new List<object>() { room }, m_ontologySettings);
+            string TTLGraph = cSharpGraph.ToTTL();
+
+            Assert.IsTrue(TTLGraph.Contains(room.BHoM_Guid.ToString()));
+
+            var res = Convert.FromTTL(TTLGraph);
+            var obj = res.Item1.First();
+            Room roomRedBack = obj as Room;
+
+            Assert.IsEqual(room, roomRedBack);
+
+        }
+
+        [Test]
+        public static void UriGuidIGeometry()
+        {
+            NurbsCurve nurbs = new NurbsCurve()
+            {
+                ControlPoints = new List<Point>()
+                {
+                    new Point(),
+                    new Point() { X = 1, Y = 1, Z = 1 },
+                    new Point() { X = 2, Y = 2, Z = 2 }
+                }
+            };
+
+            CSharpGraph cSharpGraph = Compute.CSharpGraph(new List<object>() { nurbs }, new OntologySettings()
+            {
+                TBoxSettings = new TBoxSettings { GeometryAsOntologyClass = true }
+            });
+            string TTLGraph = cSharpGraph.ToTTL();
+            IObject iObject = nurbs as IObject;
+            string hash = BH.Engine.Base.Query.Hash(iObject);
+            string nurbsGuid = Engine.Adapters.RDF.Query.GuidFromString(hash).ToString();
+
+            Assert.IsTrue(TTLGraph.Contains(nurbsGuid));
```

Comment:
This check should be replaced with:
```cs
string nurbsUrl = Flurl.Url.Combine(cSharpGraph.OntologySettings.ABoxSettings.IndividualsBaseAddress, nurbsGuid.ToString());
Assert.IsTrue(TTLGraph.Contains(nurbsUrl));
```
(will add a commit for you)

Reply:
Done in https://github.com/BHoM/RDF_Prototypes/pull/68/commits/c0f70ecdd9093e071ee5bc2837888cd65306d24e